test/Microsoft.ML.FSharp.Tests/Microsoft.ML.FSharp.Tests.fsproj

```diff
@@ -5,7 +5,7 @@
   .NET Framework 4.6.1 includes System.ValueTuple without a version, so we need to ignore the warning here.
   Bug in the latest version of VS points to a Version of FSharp.Core that doesn't exist. Temporarily ignoring
   NU1603 so it will use the latest version found.-->
-    <NoWarn>NU1603;NU1604;2003;$(NoWarn)</NoWarn>
+    <NoWarn>NU1604;2003;$(NoWarn)</NoWarn>
```

Comment:
This is an error related to System.ValueTuple, which we updated in maintenance-packages too. I am merely curious about the failure and want to see it happening. I couldn't get it to repro locally.

Reply:
I think the `NU1603` was for a version of `FSharp.Core` referenced by VS provided targets for fsproj, not ValueTuple.

Looks like `NU1604` was for value tuple https://github.com/dotnet/machinelearning/commit/791e37ed70cf8d00c6a1935c1a5764f5f12aa7f6#diff-5c7e38da31722117952bacf9106591b3ec142e24b114c5172f5d634f9bce21d1R3-R5
Which would have been caused by an old version of FSharp that referenced ValueTuple from their nuspec without listing a version.  https://github.com/dotnet/fsharp/issues/12778

I bet both have been fixed now, so lets try removing them both.